kiwi/defaults.py

```diff
@@ -181,6 +182,17 @@ def get_solvable_location():
         """
         return '/var/tmp/kiwi/satsolver'
 
+    @staticmethod
+    def set_shared_cache_location(location):
+        """
+        Sets the shared cache location once
+
+        :param str location: a location path
+        """
+        global SHARED_CACHE_DIR
+        if SHARED_CACHE_DIR == 'var/cache/kiwi':
+            SHARED_CACHE_DIR = location
```

Comment:
Thoughts about setting the cache location once ?

Reply:
did not understand the question ?

Reply:
I have no strong opinion to be honest. Probably I would have let the user just set the custom cache at any time as many times as he or she wants. I can't see why someone would like to change the location more than once and I agree that this is most likely a bad practice, but just to keep the code simple and flexible. I'd assume that someone using the API and setting custom cache location in some strange way is likely to know she or he wants.

Anyway, as I said, no strong opinion.

Reply:
are there any problems/pitfalls if cache location is set multiple times in different places ?
My guess is that there isn't any major risks because it is the cache, please let me know otherwise.